mfjcs-rest/src/main/java/org/mfjcs/core/ItemStoreImpl.java

```diff
@@ -0,0 +1,55 @@
+package org.mfjcs.core;
+
+import static org.joda.time.format.ISODateTimeFormat.basicDate;
+import static org.joda.time.format.ISODateTimeFormat.basicDateTime;
+
+import java.io.BufferedWriter;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+
+import org.mfjcs.api.Item;
+import org.mfjcs.api.ItemStore;
+import org.mfjcs.api.ItemStoreException;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+
+import io.dropwizard.jackson.Jackson;
+
+public class ItemStoreImpl implements ItemStore {
```

Comment:
Yes. Will do

On Wed, 24 Aug 2016 09:11 Marc Portier, <notifications@github.com> wrote:

> In mfjcs-rest/src/main/java/org/mfjcs/core/ItemStoreImpl.java
> <https://github.com/marc-portier/mfjcs/pull/3#discussion_r76004843>:
>
> > +
> > +import java.io.BufferedWriter;
> > +import java.io.IOException;
> > +import java.nio.file.Files;
> > +import java.nio.file.Path;
> > +import java.nio.file.Paths;
> > +
> > +import org.mfjcs.api.Item;
> > +import org.mfjcs.api.ItemStore;
> > +import org.mfjcs.api.ItemStoreException;
> > +
> > +import com.fasterxml.jackson.databind.ObjectMapper;
> > +
> > +import io.dropwizard.jackson.Jackson;
> > +
> > +public class ItemStoreImpl implements ItemStore {
>
> maybe wiser to actually upfront name the 'implementation' ?
> This being either 'default' or 'filebased', and not say S3Based or
> whatever?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/marc-portier/mfjcs/pull/3/files/a9dbe4de4a65060068a13809d530a398c07cf6e0#r76004843>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AAJVvX-VQarHJ66IyotBHRzYPoJoI0bIks5qi-6sgaJpZM4JrXRl>
> .
>